app/src/main/java/com/example/oddhours/ui/addjob/AddJobFragment.kt

```diff
@@ -60,6 +60,7 @@ class AddJobFragment : Fragment() {
             if (jobIdToEdit != null) {
                 val companyName = companyTv.text.toString().replace("'","\'").toUpperCase()
```

Comment:
we need the regex validation here as well i think. we'd want to validate input when user is saving existing job too.

maybe we can extract the functionality into a helper function?

Reply:
yup i have yet to finish the other ones, thats why i only checked off "Add Job" in the Linear ticket